common/djangoapps/django_sudo/decorators.py

```diff
@@ -0,0 +1,55 @@
+"""
```

Comment:
@dan-f I named it ```django_sudo``` because I wrote a custom ```sudo_required``` decorator instead of using default.
Yeah we can change it to ```django_sudo_helpers```.